src/core/osmLayer.js

```diff
@@ -43,6 +43,7 @@ geo.osmLayer = function (arg) {
     m_numberOfCachedTiles = 0,
     m_tileCacheSize = 100,
     m_baseUrl = 'http://tile.openstreetmap.org/',
+    m_subdomains = 'abc',
```

Comment:
> This entire file is gone in the refactored branch.

Right, should I skip testing then and create a issue instead? Also, someone should merge master in the refactored code and make sure all the new features are not lost.